src/runners/cairoRunner.test.ts

```diff
@@ -99,7 +99,7 @@ describe('cairoRunner', () => {
 
   describe('builtins', () => {
     describe('bitwise', () => {
-      test('should compute bitwise 12 & 10', () => {
+      test('should compute bitwise_ptr 12 & 10', () => {
```

Comment:
In this test you now perform all three bitwise operations, it should be renamed to reflect it

e.g. 'should compute bitwise operations 12 & 10, 12 ^10 and 12 | 10'

Reply:
Done! ✅